packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/WorkspaceSettings.tsx

```diff
@@ -76,11 +82,50 @@ export const WorkspaceSettings = () => {
   const [inviteValue, setInviteValue] = useState('');
   const [inviteLoading, setInviteLoading] = useState(false);
 
+  const [newMemberAuthorization, setNewMemberAuthorization] = React.useState<
+    TeamMemberAuthorization
+  >(
+    // get TeamMemberAuthorization key by value
+    Object.keys(TeamMemberAuthorization).find(key => {
+      const value = TeamMemberAuthorization[key];
+      return value === team?.settings.defaultAuthorization;
+    }) as TeamMemberAuthorization
+  );
+
+  const numberOfEditors = team.userAuthorizations.filter(
```

Comment:
Will `team.userAuthorizations` always exist?

Reply:
Yep, If team exists, so does team.userAuthorizations.  If you get to this page, it means team exists